docs/chaos-engineering/chaos-faults/aws/static/manifests/ec2-dns-chaos/ec2-dns-match-scheme.yaml

```diff
@@ -0,0 +1,23 @@
+# induces dns chaos on the EC2 Instances
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
+spec:
+  engineState: "active"
+  annotationCheck: "false"
+  chaosServiceAccount: litmus-admin
+  experiments:
+  - name: ec2-dns-chaos
+    spec:
+      components:
+        env:
+        # match scheme type
+        - name: MATCH_SCHEME
+          value: 'exact'
+
+        - name: EC2_INSTANCE_ID
+          value: 'instance-1'
+
+        - name: REGION
+          value: '<region for EC2_INSTANCE_ID>'
```

Comment:
Should we provide an example region name as this is an example manifest?
(At other places as well)